app/src/main/cpp/skyline/services/visrv/IApplicationDisplayService.cpp

```diff
@@ -80,6 +80,19 @@ namespace skyline::service::visrv {
         return {};
     }
 
+    Result IApplicationDisplayService::GetIndirectLayerImageRequiredMemoryInfo(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {
+        auto width{request.Pop<s64>()};
+        auto height{request.Pop<s64>()};
+        Logger::Debug("GetIndirectLayerImageRequiredMemoryInfo: width: {}, height: {}", width, height);
+
+        constexpr s64 base_size = 0x20000;
```

Comment:
`        static constexpr i64 baseSize{0x20000};`
do same for all, make sure to use camelCase